dask/base.py

```diff
@@ -156,6 +156,19 @@ def compute(self, **kwargs):
         (result,) = compute(self, traverse=False, **kwargs)
         return result
 
+    def __await__(self):
+        from distributed import wait, futures_of
```

Comment:
What happens when the user doesn't have `distributed` installed?

Reply:
The user will get an `ImportError`.  There isn't much we can do about this.  I've added a slightly more informative error message.  If you have a particular suggestion on what should happen here then please do share.

Reply:
Thanks for the fix. I think what you have pushed is good. 🙂